modules/citrus-http/src/main/java/com/consol/citrus/http/message/HttpMessage.java

```diff
@@ -339,7 +344,7 @@ public String getAccept() {
      * Gets the Http request query params.
      * @return The query parameters as a key value map
      */
-    public Map<String, String> getQueryParams() {
+    public Multimap<String, String> getQueryParams() {
```

Comment:
This breaks the public API of `HttpMessage`. But in this case, we have to break the API so that the returned value is able to represent the query parameter content regarding its specification.
Nevertheless, I'd recommend to go wit a `Map<String, Collection<String>>` to keep the guava stuff internal and have an java standard conform interface to the outer world.